test/functional/apps/visualize/_line_chart.js

```diff
@@ -221,15 +221,37 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.visEditor.changeYAxisFilterLabelsCheckbox(axisId, false);
         await PageObjects.visEditor.clickGo();
         const labels = await PageObjects.visChart.getYAxisLabels();
-        const expectedLabels = ['0', '2,000', '4,000', '6,000', '8,000', '10,000'];
+        const expectedLabels = [
```

Comment:
Did these change because of the additional space?

Reply:
Yeah, just enough vertical room at whatever display height the functional tests run at to allow for the extra ticks.